AdAway/src/main/res/values-de/strings.xml

```diff
@@ -68,99 +68,97 @@
     <string name="apply_apn_proxy_title">APN-Proxy eingestellt!</string>
-    <string name="no_connection">Die Internetverbindung funktioniert nicht.</string>
+    <string name="no_connection">Die Verbindung zum Internet funktioniert nicht.</string>
     <string name="no_connection_title">Keine Verbindung</string>
     <string name="download_dialog">Wird heruntergeladen …</string>
     <string name="apply_dialog">Wird angewendet …</string>
     <string name="apply_dialog_lists">Negativ- und Positivliste werden angewendet</string>
-    <string name="apply_dialog_hostnames">Hosts-Dateien werden verarbeitet und zusammengefügt</string>
-    <string name="apply_dialog_hosts">Hosts-Datei wird erstellt</string>
+    <string name="apply_dialog_hostnames">Analysiere und führe Hostdateien zusammen</string>
+    <string name="apply_dialog_hosts">Erstelle Host-Datei</string>
     <string name="apply_dialog_apply">Neue Hosts-Datei im System aktivieren</string>
-    <string name="apply_fail">Das Ändern der Hosts-Datei auf Ihrem System ist fehlgeschlagen!\n
-  Versuchen Sie bitte die Hosts-Datei, in den Einstellungen, auf /data/data/hosts zu ändern.</string>
+    <string name="apply_fail">Anwenden der Host-Datei auf Ihr System fehlgeschlagen! Versuchen Sie, die Zielhostdatei in den Einstellungen in /data/data/hosts zu ändern.</string>
     <string name="apply_fail_title">Anwendung fehlgeschlagen</string>
-    <string name="revert_successful">Die Änderungen wurden rückgängig gemacht.\n
-  Damit alle Änderungen wirksam werden, muss das System eventuell neu gestartet werden.\n
-  Möchten Sie neu starten?\n
-  (Um Datenverlust zu vermeiden, vergewissern Sie sich, dass keine Anwendung gerade die SD-Karte verwendet!)</string>
+    <string name="revert_successful">Sie haben die Standard-Hosts-Datei wiederhergestellt. Es wird empfohlen, Android Neu zu Starten, damit die Änderungen wirksam werden. \n
+    \n
+  Möchten Sie Neu Starten? (Um Datenverlust zu vermeiden, vergewissern Sie sich, dass keine App die Speicherkarte nutzt!)</string>
     <string name="revert_successful_title">Erfolgreich zurückgesetzt</string>
-    <string name="revert_problem">Wiederherstellen fehlgeschlagen.</string>
-    <string name="revert_problem_title">Die Wiederherstellung ist fehlgeschlagen!</string>
-    <string name="download_fail_dialog">Keine der aktivierten Hosts-Quellen ist erreichbar! Funktioniert Ihre Internetverbindung richtig?</string>
+    <string name="revert_problem">Das Zurücksetzen ist aus unbekannten Gründen fehlgeschlagen.</string>
+    <string name="revert_problem_title">Zurücksetzen fehlgeschlagen!</string>
+    <string name="download_fail_dialog">Keine der aktivierten Hostquellen ist erreichbar! Sind Sie richtig mit dem Internet verbunden?</string>
```

Comment:
Would delete `richtig`. The relevant information for the user is that there might be a problem with his internet-connection.